mitosheet/src/components/endo/FormulaBar.tsx

```diff
@@ -44,7 +66,38 @@ const FormulaBar = (props: {
                 <div className="formula-bar-vertical-line"/>
             </Col>
             <Col flex='1'>
-                <input className="formula-bar-formula text-header-3 text-overflow-hide element-width-block" value={formulaBarValue} disabled />
+                {props.editorState?.editorLocation === 'formula bar' &&
+                    <CellEditor
+                        sheetData={props.sheetData}
+                        sheetIndex={props.sheetIndex}
+                        gridState={props.gridState}
+                        editorState={props.editorState}
+                        setEditorState={props.setEditorState}
+                        setGridState={props.setGridState}
+                        setUIState={props.setUIState}
+                        scrollAndRenderedContainerRef={props.scrollAndRenderedContainerRef}
+                        containerRef={props.containerRef}
+                        mitoAPI={props.mitoAPI}
+                        currentSheetView={currentSheetView}
+                    />
+                } 
+
+                {(props.editorState === undefined || props.editorState.editorLocation === 'cell') &&
```

Comment:
I think we can do the inverse of the check above? Just `props.editorState?.editorLocation !== 'formula bar'` - rather than this more complicated check?